contracts/libraries/MakerTraitsLib.sol

```diff
@@ -76,6 +76,15 @@ library MakerTraitsLib {
         return expiration != 0 && expiration < block.timestamp;  // solhint-disable-line not-rely-on-time
     }
 
+    /**
+     * @notice Returns the expiration time of the order.
+     * @param makerTraits The traits of the maker.
+     * @return result The expiration timestamp of the order.
+     */
+    function getExpirationTime(MakerTraits makerTraits) internal pure returns (uint256) {
```

Comment:
Can it be used in [line 75](https://github.com/1inch/limit-order-protocol/blob/ad7c6cbeca19188eb0176820c7715ba89f4311b9/contracts/libraries/MakerTraitsLib.sol#L75)?

Reply:
fixed in [2e60afc](https://github.com/1inch/limit-order-protocol/pull/358/commits/2e60afc3b5d78ac8de2237c83714c012861debba)